com.github.Try.OpenGothic.json

```diff
@@ -0,0 +1,62 @@
+{
+	"app-id": "com.github.Try.OpenGothic",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"finish-args": [
+		"--socket=x11",
+		"--socket=pulseaudio",
+		"--device=dri",
+		"--filesystem=host"
+	],
+	"modules": [
+		{
+			"name": "opengothic",
+			"buildsystem": "cmake",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/Try/OpenGothic/archive/refs/tags/v0.37.tar.gz",
```

Comment:
What's the reason to use tarball for the main repo and pick git modules individually? Why not just use git source here?

Reply:
No reason, really. I'll change it.